src/Online_Bookstore/ReviewSection/ReviewedBooks.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import ReviewCard from "./ReviewCard";
+const ReviewedBooks = () => {
+  const [reviewData, setReviewData] = useState([]);
+  const navigate = useNavigate();
+  const UserData = JSON.parse(window.localStorage.getItem("user_login"));
```

Comment:
move the implementation of local storage (get and set) in a separate file, return the required data, and use those in the components. Avoid the repetition of writing local storage again and again in each component.